ocs_ci/ocs/ui/base_ui.py

```diff
@@ -946,23 +963,20 @@ def login_ui(console_url=None, username=None, password=None):
     driver.get(console_url)
     # Validate proceeding to the login console before taking any action:
     proceed_to_login_console(driver)
-    if config.ENV_DATA.get("flexy_deployment") or config.ENV_DATA.get(
-        "import_clusters_to_acm"
-    ):
-        try:
-            element = wait.until(
-                ec.element_to_be_clickable(
-                    (
-                        login_loc["kubeadmin_login_approval"][1],
-                        login_loc["kubeadmin_login_approval"][0],
-                    )
+    try:
+        element = wait.until(
+            ec.element_to_be_clickable(
+                (
+                    login_loc["kubeadmin_login_approval"][1],
+                    login_loc["kubeadmin_login_approval"][0],
```

Comment:
Can you pls explain these lines? What are we doing here?

Reply:
After you create a user on Openshift, you get this login page 
![image](https://user-images.githubusercontent.com/61982127/211334254-0837d6a4-1741-4414-80e7-2d6b2a9be8cb.png)
and we need to click on `kube:admin`